File: sanidad-backend/src/main/java/com/farmacia/sanidadbackend/dto/SugerenciaProductoDTO.java
package com.farmacia.sanidadbackend.dto;

import lombok.Data;

import java.math.BigDecimal;

@Data
public class SugerenciaProductoDTO {
    private Long medicamentoId;
    private String nombre;
    private String presentacion;
    private BigDecimal precioUnitario;
    private String tipoSugerencia; // "FIFO", "COMPLEMENTARIO", "FRECUENTE_CLIENTE"
    private String mensaje;
    private Long loteDetalleId; // Para FIFO (opcional)
    private Integer cantidadSugerida; // Opcional
}

File: SanidadMovil/app/src/main/java/com/sanidad/movil/data/remote/dto/SugerenciaProductoDTO.kt
package com.sanidad.movil.data.remote.dto

data class SugerenciaProductoDTO(
    val medicamentoId: Long,
    val nombre: String,
    val presentacion: String,
    val precioUnitario: Double,
    val tipoSugerencia: String,
    val mensaje: String,
    val loteDetalleId: Long?,
    val cantidadSugerida: Int?
)